inquirer/render/console/_list.py

```diff
@@ -45,10 +45,10 @@ def get_options(self):
                 or (is_in_end and end_index == self.current)
             ):
 
-                color = self.theme.List.selection_color
-                symbol = self.theme.List.selection_cursor
+                color = self.theme.list.selection_color
+                symbol = self.theme.list.selection_cursor
             else:
```

Comment:
list is a reserved word on Python. This can lead to problems, would be better if it renamed to something else.